extensions/common/http/jersey-core/src/main/java/org/eclipse/edc/web/jersey/mapper/EdcApiExceptionMapper.java

```diff
@@ -50,7 +50,7 @@ public EdcApiExceptionMapper() {
                 NotAuthorizedException.class, FORBIDDEN,
                 InvalidRequestException.class, BAD_REQUEST,
                 ObjectNotFoundException.class, NOT_FOUND,
-                ObjectExistsException.class, CONFLICT,
+                ObjectConflictException.class, CONFLICT,
                 ObjectNotModifiableException.class, CONFLICT,
```

Comment:
while you are at it, this `ObjectNotModifiableException` class could be removed, it's not used by anyone

Reply:
:axe: :+1: